src/panels/config/core/ha-config-section-core.js

```diff
@@ -30,6 +31,7 @@ class HaConfigSectionCore extends LocalizeMixin(PolymerElement) {
         <ha-config-name-form hass="[[hass]]"></ha-config-name-form>
         <ha-config-core-form hass="[[hass]]"></ha-config-core-form>
         <ha-config-url-form hass="[[hass]]"></ha-config-url-form>
+        <ha-config-network hass="[[hass]]"></ha-config-network>
```

Comment:
I think that this one (and the one above it) should both be hidden unless a user has advanced mode enabled.

Reply:
Url is already hidden unless they are in advanced mode.

I'll put the same change in network

Reply:
97a8aa9f0